backend/djangocookieauth/settings.py

```diff
@@ -117,28 +120,56 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
-STATIC_URL = 'static/'
+STATIC_URL = "static/"
 
 # Default primary key field type
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
-DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
-CSRF_COOKIE_SAMESITE = 'Strict'
-SESSION_COOKIE_SAMESITE = 'Strict'
+CSRF_COOKIE_SAMESITE = "Strict"
+SESSION_COOKIE_SAMESITE = "Strict"
 CSRF_COOKIE_HTTPONLY = True
 SESSION_COOKIE_HTTPONLY = True
-CSRF_TRUSTED_ORIGINS = ['http://localhost:81']
+CSRF_TRUSTED_ORIGINS = ["http://localhost:81"]
```

Comment:
csrf are not needed anymore once JWT are used 
tested with 
token="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0b2tlbl90eXBlIjoicmVmcmVzaCIsImV4cCI6MTcxNzA4NjcxMCwiaWF0IjoxNzE3MDAwMzEwLCJqdGkiOiIzOTBmNjczNWZmNTY0ODIyOTI0NGJlYjhmM2I1ZGY0NCIsInVzZXJfaWQiOjJ9.n4HFgmLFZ9ZNHr8Z6XRqsjwzMejZt0uSh8L_yPGDA_g"

curl http://localhost:81/api/whoami/ -H "Authorization: Bearer $token"

token is access token